### ISSUE TITLE
About the training parameters of hrnet18 baseline model

### ISSUE THREAD

**Issue by whut2962575697** [ifzhang/FairMOT#57]
Hello, thank you for your great job! I see that you have updated a baseline model of hrnet18 that reaches 76.6 on mot15. Is it convenient to provide training parameters, such as the number of iterations and learning rate settings? In addition, if I want to further fine-tune it on mot15, is the learning rate and the number of iterations the same as in the DAL34 fine-tuning script you gave?

**Comment by ifzhang** [comment]
When I train hrnet18 on the large training data, I set batchsize=4 and train 60 epochs. HRNet tends to perform better when training more epochs. When finetuning on MOT15, the parameters can be same as dla34.
